stories.html

```diff
@@ -0,0 +1,90 @@
+<html>
+   <head>
+            <title>Latest Stories</title>
+            <style>
+                body{
+                    margin-top: 0px;
+                     margin-right: 0px;
+                     margin-bottom: 0px;
+                    margin-left: 0px;
+                }
+                ul{
+                    padding-inline-start: 1px;
+                    
+                }
+                ul>li{
+                    list-style-type: none;
+                    float:left;
+                    padding-right: 10px;
+                    
+                   
+                }
+                p{
+                    font-weight: bold;
+                    font-size:18px;
+                    font-family: 'Times New Roman';
+                    text-align: justify;
+                    padding-top: 0px;
+                   
+                }
+                
+                .vertical { 
```

Comment:
instead of creating a new style for this, you can use the pseudo selector last-child and remove the border. 
https://css-tricks.com/almanac/selectors/l/last-child/

Reply:
added a pseudo class selector